public/style.css

```diff
@@ -66,29 +77,49 @@ div {
 	width: auto;
 }
 
+.review-details-box {
+  width: 730px;
+}
+
 .title {
   margin: 20px 0;
-	line-height: 1.2;
+  line-height: 1.2;
   font-size: 25px;
   font-weight: 100;
 }
 
 .details {
-	font-weight: 100;
+  font-weight: 100;
   line-height: 1.3;
   margin-bottom: 25px;
   color: #666666;
 }
 
-.prefix {
+.review-interactor {
+  width: 730px;
+  height: auto;
+  padding-top: 25px;
+  background-image: url(https://lululemon.ugc.bazaarvoice.com/static/7834redes/dottedBorder.gif);
+  background-position-x: left;
+  background-position-y: top;
+  background-size: initial;
+  background-repeat-x: repeat;
+  background-repeat-y: no-repeat;
+  background-attachment: initial;
+  background-origin: initial;
+  background-clip: initial;
+  background-color: transparent;
+}
+
+.review-interactor-prefix {
 	float: left;
   line-height: 1;
   padding-right: 20px;
   letter-spacing: normal;
```

Comment:
Are you overriding some styling you created earlier with line-height and letter-spacing? If not you can remove these lines because they are defaults.